sim/global-types.ts

```diff
@@ -64,7 +64,7 @@ namespace TierTypes {
 	export type Singles = "AG" | "Uber" | "(Uber)" | "OU" | "(OU)" | "UUBL" | "UU" | "RUBL" | "RU" | "NUBL" | "NU" |
 	"(NU)" | "PUBL" | "PU" | "(PU)" | "NFE" | "LC";
 	export type Doubles = "DUber" | "(DUber)" | "DOU" | "(DOU)" | "DBL" | "DUU" | "(DUU)" | "NFE" | "LC";
-	export type Other = "Unreleased" | "Illegal" | "CAP" | "CAP NFE" | "CAP LC";
+	export type Other = "Unreleased" | "Illegal" | "CAP" | "CAP NFE" | "CAP LC" | "NS" | "CE" | "MnM";
```

Comment:
@KrisXV Continuing from discord: Is there another place we can put these special tier values? It seems dangerous to have values that are outside of the enum types. Maybe another enum called `Special` or something?

Reply:
These are never actually used for tiering and the reason for declaring types for `tier` etc is to ensure there's no typos when editing data in `formats-data` files. We could do an enum specific to the OM chat file or something, but these shouldn't be here.

Reply:
I see. I have an idea, will give it a shot

Reply:
How's the latest rev? Keeps the values in that file

Reply:
That's fine